src/system/SystemLayer.h

```diff
@@ -288,6 +290,32 @@ class LayerSockets : public Layer
     virtual SocketWatchToken InvalidSocketWatchToken() = 0;
 };
 
+class LayerSocketsLoop;
+
+/**
+ * Enables the participation of subordinate event loops in the SystemLayer event loop.
```

Comment:
Needs more documentation. What is a subordinate event loop? What are usage examples?

Reply:
I've updated the documentation.